software/soapysdr-xtrx/XTRXDevice.hpp

```diff
@@ -309,6 +309,10 @@ class DLL_EXPORT SoapyXTRX : public SoapySDR::Device {
     void writeUART(const std::string &which, const std::string &data) override;
     std::string readUART(const std::string &which, const long timeoutUs) const override;
 
+    // Return the internal LMS7 device handle, allowing the caller to bypass us and
+    // directly call LMS7002M-driver code.
+    void * getLMS7Handle();
```

Comment:
You can also use the SoapySPI for this: https://github.com/pothosware/SoapySDR/blob/637023d5e1b60e117a6533daddc4d991a33375f9/include/SoapySDR/Device.hpp#L1450